build/generate-images.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+'use strict';
+
+const imageminZopfli = require('imagemin-zopfli'); // eslint-disable-line import/no-unresolved
+const svg2img = require('svg2img'); // eslint-disable-line import/no-unresolved
+const {DOMParser, XMLSerializer} = require('xmldom'); // eslint-disable-line import/no-unresolved
+const {promisify} = require('util');
+const {readFile, writeFile} = require('fs').promises;
+const {resolve} = require('path');
+
+function exit(err) {
+  if (err) console.error(err);
+  process.exit(err ? 1 : 0);
+}
+
+async function generate(svg, outputFile, {size, bg, removeDetail} = {}) {
+  const parser = new DOMParser();
+  const serializer = new XMLSerializer();
+  const document = parser.parseFromString(svg);
+
+  if (bg) {
+    const rect = parser.parseFromString('<rect width="100%" height="100%" fill="white"/>');
+    document.documentElement.insertBefore(rect, document.documentElement.firstChild);
+  }
+
+  if (removeDetail) {
+    for (const el of Array.from(document.getElementsByTagName('g') || [])) {
+      for (const attribute of Array.from(el.attributes || [])) {
+        if (attribute.name === 'class' && attribute.value === 'detail-remove') {
```

Comment:
Why not just use id attribute and layer3 as value?

Reply:
I added that class specifically so users can run their own SVGs through it and `layer3` is specific to our source image so could destroy parts of the user image.

Reply:
Also, added this info about `detail-remove` to the docs.